service/src/main/java/uk/gov/hmcts/reform/fpl/config/LocalAuthorityCodeLookupConfiguration.java

```diff
@@ -19,7 +20,7 @@ public LocalAuthorityCodeLookupConfiguration(@Value("${fpl.local_authority_email
     }
 
     public String getLocalAuthorityCode(String emailDomain) {
-        checkNotNull(emailDomain, "Email domain cannot be null");
```

Comment:
deprecated in assertj version required by spring 2.3.0, anyway it was from testing support used in prod code

Reply:
Maybe it was intended to use the [guava preconditions](https://github.com/google/guava/wiki/PreconditionsExplained), it has the same method name

Reply:
could be, anyway prefer use native java counterpart